docs/index.md

```diff
@@ -2,6 +2,42 @@
 
 ### Description
 
+### Manual:
+To get a model for a SAT formula we use `get-value ()` and specify the desired terms for within that: For e.g,  
+`(get-value (x y))` will print the value for x and y.
+
+
+**Interpolation:**
+LRA interpolation algorithms can be changed with the follwing option in the command line:
+
+`(set-option :interpolation-lra-algorithm <N>)`
+where N can be:
+- 0 for Farkas interpolation
+- 2 for dual Farkas interpolation
+- 4 for decomposed interpolants
+- 5 for dual decomposed interpolants
+- 3 for no-man's land algorithm
+
```

Comment:
Maybe give references here to the HVC 2017 and TACAS 2019 papers where the algorithms are defined